jobs/payment-jobs/tasks/ejv_partner_distribution_task.py

```diff
@@ -389,17 +389,21 @@ def get_partner_disbursements(
                 DistributionCodeModel,
                 DistributionCodeModel.distribution_code_id == PaymentLineItemModel.fee_distribution_id,
             ).filter(
-                PartnerDisbursementsModel.status_code == DisbursementStatus.WAITING_FOR_JOB.value,
-                PartnerDisbursementsModel.partner_code == partner.code,
+                PartnerDisbursementsModel.status_code == DisbursementStatus.WAITING_FOR_JOB.value
+            ).filter(
+                PartnerDisbursementsModel.partner_code == partner.code
+            ).filter(
+                DistributionCodeModel.stop_ejv.is_(False) | DistributionCodeModel.stop_ejv.is_(None)
+            ).filter(
+                ~InvoiceModel.receipts.any(cast(ReceiptModel.receipt_date, Date) >= disbursement_date.date())
+            ).filter(
```

Comment:
I'm trying to get this part exact same as before, 
![image](https://github.com/user-attachments/assets/3590dbd1-66c1-4551-bee0-017a7024d455)
https://github.com/bcgov/sbc-pay/pull/1945/files#diff-49daea3b47b0d4a84328e1fbb734e6287d8babaf3ef3cca83b07eb37fa162ea9